CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # March 2021, the oldest we have to support : Ununtu 18.04 LTS
-cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.11 FATAL_ERROR)
```

Comment:
For FreeCAD 1.0 we require CMake 3.16, see https://wiki.freecadweb.org/FreeCAD_1.0_Development_Cycle